.github/workflows/zxc-e2e-test.yaml

```
@@ -179,7 +179,7 @@ jobs:
         run: docker image pull kindest/node:v1.31.4@sha256:2cb39f7295fe7eafee0842b1052a599a4fb0f8bcf3f83d96c7f4864c357c6c30
 
       - name: Setup Java
-        if: ${{ runner.os == 'linux' && (inputs.test-script == 'test-e2e-node-local-hedera' || inputs.test-script == 'test-e2e-node-local-ptt' || inputs.test-script == 'test-e2e-node-add-local') && !cancelled() && !failure() }}
+        if: ${{ runner.os == 'linux' && (inputs.test-script == 'test-e2e-node-local-hedera' || inputs.test-script == 'test-e2e-node-local-ptt' || inputs.test-script == 'test-e2e-node-add-local' || inputs.test-script == 'test-e2e-block-node') && !cancelled() && !failure() }}
```

```suggestion
        if: ${{ runner.os == 'linux' && inputs.local-java-build && !cancelled() && !failure() }}
```
Let us add a new `local-java-build` property onto the objects in the matrix JSON, it will clean this up and make it easier to maintain going forward.

---

added